### PR TITLE
New notification type for boss damage

### LINKED ISSUE #12221: new design for boss damage notifications from tasks and skills
As stated by @Tressley at https://github.com/HabitRPG/habitica/issues/9136#issuecomment-575685571 :

"Here's a Zeplin artboard with how damage should be displayed: https://zpl.io/2p0d882
image
"

This design should be applied to the notification you see when you do damage to the boss by completing a task or by casting a skill (Burst of Flames, Brutal Smash).

To get an invitation to the Zeplin artboard to see the details of the design, please tell us your email address. If you don't want to post it here publicly, email admin@habitica.com and include the URL of this issue - however post here first to claim the issue so that someone else doesn't start on it!

### PR DESCRIPTION
**Pull request by cvuorinen**
[//]: # (Note: See http://habitica.fandom.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #12221

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Adds new notification type for boss damage with sword icon.
Changed translation key since the old text doesn't work as there is no value inserted into the text anymore.

Screenshot from 2020-05-25 23-18-11

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: fa756acf-9127-4a3c-8dac-8ff627d30073

### CODE REVIEW

File: website/client/src/components/snackbars/notification.vue [hunk 1]
```diff
@@ -133,6 +148,10 @@
     color: #2995cd;
   }
 
+  .damage .icon {
+    color: #686274;
```
Comment (paglias) [inline website/client/src/components/snackbars/notification.vue]: I know this isn't used much in this component, but could you switch from the raw color value to the variable `$gray-100`? You'll have to import the colors variables at the top `@import '~@/assets/scss/colors.scss';`

**Review by paglias — CHANGES_REQUESTED** [review]
@cvuorinen left a small comment but it's working well and I'll merge it as soon as it's handled, thanks!

File: website/client/src/components/snackbars/notification.vue (hunk 1 above)
Comment (cvuorinen) [reply website/client/src/components/snackbars/notification.vue]: I'll change the other colors to use vars too in the next PR, where there will be more changes to the styling.

**Comment by paglias** [comment]
thanks @cvuorinen !

**Pull request merged**